src/index.js

```diff
@@ -102,6 +104,12 @@ async function writePackageJSON() {
   await writeJsonFile(path, pkg, { detectIndent: true });
 }
 
+async function installDeps() {
+  const command = hasYarn() ? 'yarn' : 'npm';
```

Comment:
Hi, would it be better to do `hasYarn(cwd)` instead so that we can guarantee to check the current working dir?

Reply:
It looks like it defaults to cwd already https://github.com/sindresorhus/has-yarn#cwd